### PR TITLE
Settings rearranged by type

### PR DESCRIPTION
**Pull request by Pranav-FP**

### CODE REVIEW

**Comment by Pranav-FP** [comment]
- Rearranged into 4 types - In level, Player Action, Boss, Quest
- Added Tooltips to explain some splits
- Some minor text changes (spelling, etc.)

File: POPTLC.Settings.xml [hunk 1]
```diff
@@ -33,28 +49,26 @@
...
         <Setting Id="boss__7056__CIT_LOW_22_Radjen" Label="Ardashir" State="false" />
         <!-- <Setting Id="boss__7413__270" Label="Alternate Sargon (Sacred Archives)" State="false" /> -->
         <Setting Id="boss__7413__HYR_SOM_22_CorruptedTwinSargon" Label="Alternate Sargons (Hyrcanian Forest)" State="false" />
         <Setting Id="boss__7413__END_PIT_15_CorruptedSargon" Label="Alternate Sargon (Pit of Eternal Sand)" State="false" />
     </Setting>
-
+    
     <Setting Id="quest" Label="Split on completing a quest" State="true">
-        <Setting Id="inlevel_OLD_01" Label="Persia Under Attack (Battlefield + Thomyris Palace)" State="false" />
+        <Setting Id="quest_end_e36c64cd5a982ce4b8571f8190c1ad43" Label="Persia Under Attack" State="false" ToolTip="Splits on finishing Thomyris Palace (Unreliable)" />
```
Comment (mitchell-merry) [inline POPTLC.Settings.xml:66]: FYI this is fundamentally broken, we should not include this quest split

This is why those level splits exist, they make these quest splits reliable

File: POPTLC.Settings.xml [hunk 2]
```diff
@@ -73,31 +87,31 @@
...
+        <Setting Id="quest_end_89a9bdf53da19a744b22fad494884202" Label="The Tiger and the Rat" State="false" ToolTip="Splits on talking to old man after getting shadow" />
         <!-- 
             89a9bdf53da19a744b22fad494884202
 
             Chase after the Old Man in the Depths of the Citadel to get your Eye back
          -->
 
          <!-- Quest sometimes doesn't end correctly -->
-        <Setting Id="inlevel_PIT_01" Label="The Path to the Sand Prison ($10,000 Challenge End)" State="false" />
+        <Setting Id="quest_end_e499d16e671ee1b42b170dc0f71b6c8e" Label="The Path to the Sand Prison" State="false" ToolTip="Splits on entering sand pit (Unreliable)" />
```
Comment (mitchell-merry) [inline POPTLC.Settings.xml:98]: This is also fundamentally broken

File: POPTLC.Settings.xml (hunk 1 above)
Comment (mitchell-merry) [reply POPTLC.Settings.xml:66]: The same goes for the Vahram 3 QTE - it's to make the vahram 3 boss kill reliable.

And the same could go for menolias & orod QTE if we prefer it to split then. But we should pick just one.

**Review by mitchell-merry — COMMENTED** [review]
My general vibe is: player action & level splits are, at the moment, patching up bad quest splits (unreliable quests), and getting better split points for the boss fights.

Speedrunners don't care about what the ASL uses to actually split on these conditions, they just care if they know when it does. I also think we should de-deduplicate these things.

File: POPTLC.Settings.xml (hunk 2 above)
Comment (Pranav-FP) [reply POPTLC.Settings.xml]: Removed from settings.

File: POPTLC.Settings.xml (hunk 1 above)
Comment (Pranav-FP) [reply POPTLC.Settings.xml]: Removed `quest_end_e36c64cd5a982ce4b8571f8190c1ad43` from settings.

**Pull request merged**